risc0/circuit/bigint/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name = "risc0-circuit-bigint"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0"
+bytemuck = "1.16.1"
+clap = { version = "4.1", features = ["derive"] }
```

Comment:
I think this is a `dev-dependency`?

Reply:
Nope, but we can put it behind a feature